eventmesh-common/src/main/java/org/apache/eventmesh/common/protocol/SubscriptionItem.java

```diff
@@ -66,6 +66,11 @@ public String toString() {
                 + ", type=" + type
                 + '}';
     }
+
+    @Override
+    public SubscriptionItem clone() {
```

Comment:
I suggest implementing Serializable interface to implement deep clone instead of overriding Cloneable interface.